open-company/open-company-web
# UX desktop: invite notification

Card: https://trello.com/c/Qj35yZPt

To test:
- go to invite panel
- insert an invite via email
- send it
- [x] do you see notification saying "Invite sent"? Good
- now insert a bunch of invites (mixed with Slack if you want)
- send them
- [x] do you see a notification saying "Invites sent"? Good
- now insert try to invite an email already registered (use your for example)
- [x] did you NOT get the notification? Good
